test/acceptance/coffee/CRUDTests.coffee

```diff
@@ -0,0 +1,201 @@
+{expect} = require('chai')
+mongojs = require("mongojs")
+{ObjectId} = mongojs
+
+Settings = require('settings-sharelatex')
+db = mongojs(Settings.mongo.url, ['tags'])
+
+TagsApp = require './helpers/TagsApp'
+TagsClient = require './helpers/TagsClient'
+
+describe 'CRUDTests', ->
+	before (done) ->
+		db.tags.createIndex({
+				user_id: 1,
+				name: 1
+			}, {
+				unique: true,
+				key: {
+					user_id: 1,
+					name: 1
+				},
+				name: 'user_id_1_name_1'
+			}, done)
+
+	before (done) ->
+		TagsApp.ensureRunning done
+
+	beforeEach =>
```

Comment:
I think the mocha functions should be using `->` instead of fat arrows. Mocha has it's "this pointer" magic, so in general we should have `->` for `describe`, `before/after[Each]` and `it`, and then fat arrows for callbacks within the methods.